### PR TITLE
add email and complete footer section

### PR DESCRIPTION
**Pull request by MarsIncarnate**
This PR completes the footer section with an email address and background image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,10 +231,52 @@ <h4>About me</h4>
         </p>        
         <form action="">
```
Comment (muneebulrehman) [inline index.html]: Your form should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).

- `action` value should be similar to https://formspree.io/f/{form_id}

- `method` value equals POST

File: index.html [hunk 2]
```diff
@@ -231,10 +231,52 @@ <h4>About me</h4>
...
             <input type="text" name="full_name" id="" placeholder="Full Name" required>
```
Comment (muneebulrehman) [inline index.html]: Name input should have a `maxlength` attribute and `maxlength` should be 30.

File: index.html [hunk 3]
```diff
@@ -231,10 +231,52 @@ <h4>About me</h4>
...
-            <input type="email" name="" id="" placeholder="Email Address" required>
-            <textarea name="" id="" rows="5" placeholder="Enter Text Here"></textarea>
+            <input type="email" name="email" id="" placeholder="Email Address" required>
+            <textarea name="message" id="" rows="5" placeholder="Enter Text Here"></textarea>
```
Comment (muneebulrehman) [inline index.html]: `textarea` should have both `maxlength` and `required` attributes with `maxlength`  equal to 500.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @MarsIncarnate , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Followed design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @MarsIncarnate   👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**